docs/report/pack.md

```diff
@@ -52,3 +52,20 @@ that is needed for the build. This will result in the following printout:
 
 {% with id="report-options", url="report/picklist.png", description="Picklist Example" %} {% include "img.html" %} {% endwith %}
 
+For those of you who would like to replace the "/" by something else because it is hard 
+the read in certain fonts use the following trick:
```

Comment:
@SergeoLacruz not sure what you mean here about "fonts"?

Reply:
@SchrodingersGat probably font as in font face/font family.